src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -1,450 +0,0 @@
-                "legal_basis_for_processing"
-            ),
-            PrivacyDeclaration.features,
-            PrivacyDeclaration.retention_period,
-            PrivacyDeclaration.purpose,
-            PrivacyDeclaration.legal_basis_for_processing.label(
-                "original_legal_basis_for_processing"
-            ),
-        )
-        .outerjoin(PrivacyDeclaration, System.id == PrivacyDeclaration.system_id)
-        .outerjoin(
-            legal_basis_override_subquery,
-            legal_basis_override_subquery.c.id == PrivacyDeclaration.id,
-        )
-        .filter(
-            or_(
-                and_(GVL_DATA_USE_FILTER, consent_legal_basis_filter),
-                and_(
-                    GVL_DATA_USE_FILTER,
-                    legitimate_interest_legal_basis_filter,
-                    NOT_AC_SYSTEM_FILTER,
-                ),
-                AC_SYSTEM_NO_PRIVACY_DECL_FILTER,
-            )
-        )
-        .order_by(
-            PrivacyDeclaration.created_at.desc()
-        )  # Order to get repeatable results when collapsing information
-    )
-    if not CONFIG.consent.ac_enabled:
-        # If AC Mode is not enabled, exclude all Privacy Declarations tied to an AC-based system
-        matching_privacy_declarations = matching_privacy_declarations.filter(
-            NOT_AC_SYSTEM_FILTER
-        )
-
-    return (
-        matching_privacy_declarations,
-        consent_legal_basis_filter,
-        legitimate_interest_legal_basis_filter,
-    )
-
-
-def systems_that_match_tcf_data_uses(
-    matching_privacy_declarations: Query, data_uses: List[str]
-) -> List[FidesKey]:
-    """Check which systems have these data uses directly.
-
-    This is used for determining relevant systems for TCF purposes and special purposes. Unlike
-    determining relevant systems for Privacy Notices where we use a hierarchy-type matching,
-    for TCF, we're looking for an exact match on data use."""
-    if not data_uses:
-        return []
-
-    return list(
-        {
-            privacy_declaration_record.system_fides_key
-            for privacy_declaration_record in matching_privacy_declarations.filter(
-                PrivacyDeclaration.data_use.in_(data_uses)
-            )
-        }
-    )
-
-
-def get_relevant_systems_for_tcf_attribute(  # pylint: disable=too-many-return-statements
-    db: Session,
-    tcf_field: Optional[str],
-    tcf_value: Union[Optional[str], Optional[int]],
-) -> List[FidesKey]:
-    """Return a list of the system fides keys that match the given TCF attribute,
-    provided the system is relevant for TCF.
-
-    Used for consent reporting, to take a snapshot of the systems that are relevant at that point in time
-    """
-
-    # For TCF attributes, we need to first filter to systems/privacy declarations that have a relevant GVL data use
-    # as well as a legal basis of processing of consent or legitimate interests
-    (
```

Comment:
This hard-to-maintain functionality that calculated relevant systems for TCF for consent reporting has been removed altogether and is not a part of the new consent reporting table.